source/_patterns/02-organisms/teaser-list/teaser-list.twig

```diff
@@ -0,0 +1,54 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser-list']) }) -%}
+
+<div {%- include('atoms-attributes') -%}>
+
+  {%- with title|default({}) only -%}
+    {%- block title -%}
+
+      {% if text is defined %}
+
+        {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['teaser-list__title']) }) -%}
+        {%- include 'atoms-heading' with {level: level|default(4)} -%}
```

Comment:
Probably want a way to link the `title` and the `list`. How about generating an ID for the `title` if it doesn't already exist, and use that as an `aria-labelledby` on the wrapping `div`?